### PR TITLE
#159 Update DG with Multi-day Order feature

### LINKED ISSUE #159: Update DG with Multi-day Order feature

### PR DESCRIPTION
**Pull request by Cazh1**
Fixes #159

### CODE REVIEW

File: docs/images/class/PreviousDayCommand.png
Comment (ziyi105) [inline docs/images/class/PreviousDayCommand.png:1]: Why did you underline scanner? It is not a static method

Comment (ziyi105) [inline docs/images/class/PreviousDayCommand.png:1]: Perhaps you can add explanation about the association. E.g., CafeCtrl executes Command

**Review by ziyi105 — APPROVED** [review]

File: docs/images/class/PreviousDayCommand.png
Comment (Cazh1) [reply docs/images/class/PreviousDayCommand.png:1]: Please elaborate on how to add this explaination

Comment (ziyi105) [reply docs/images/class/PreviousDayCommand.png:1]: hmm CafeCtrl calls execute() in command class, so we can explain their association as CafeCtrl executes Command, and maybe you can add execute beside the arrow pointing from CafeCtrl to Command?

**Pull request merged**